spk/git/Makefile

```
@@ -29,6 +29,9 @@ SPK_COMMANDS += bin/git-shell
 SPK_COMMANDS += bin/git-upload-archive
 SPK_COMMANDS += bin/git-upload-pack
 SPK_COMMANDS += bin/scalar
+
+ifneq ($(findstring $(ARCH),$(PPC_ARCHS) $(ARMv5_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
 SPK_COMMANDS += bin/git-lfs
+endif
```

It is not a problem to add a command that not exist to SPC_COMMANDS, it will be simply ignored. But for the sake of cleanness you are right.

But let's reduce the arch evaluation and include the optional dependency into this (and remove it above)

```suggestion
ifneq ($(findstring $(ARCH),$(PPC_ARCHS) $(ARMv5_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
DEPENDS += cross/git-lfs
SPK_COMMANDS += bin/git-lfs
endif
```